controllers/releaseplan/adapter.go

```diff
@@ -102,3 +103,21 @@ func (a *adapter) EnsureMatchingInformationIsSet() (controller.OperationResult,
 
 	return controller.RequeueOnErrorOrContinue(a.client.Status().Patch(a.ctx, a.releasePlan, patch))
 }
+
+// EnsureReleaseGracePeriodDaysIsSet is an operation that will ensure that the ReleasePlan has the
+// ReleaseGracePeriodDays set
+func (a *adapter) EnsureReleaseGracePeriodDaysIsSet() (controller.OperationResult, error) {
```

Comment:
I'm not sure what we need this for since this is already done in the webhook

Reply:
Maybe I did something wrong, but In my test the value was added the on creation through the webhook but after reconciling it lost the value. I will test once again without it.

Reply:
Removed.